File: AppjedrezApi/AuthApi/Models/Inscripcion.cs
﻿using System;
using System.Collections.Generic;

namespace AuthApi.Models;

public partial class Inscripcion
{
    public Guid Id { get; set; }

    public Guid? IdTorneo { get; set; }

    public Guid? IdParticipante { get; set; }

    public TimeSpan? HoraInscripcion { get; set; }

    public string Estado { get; set; }

    public DateTime? Fecha { get; set; }

    public virtual Usuario IdParticipanteNavigation { get; set; }

    public virtual Torneo IdTorneoNavigation { get; set; }
}

File: appjedrez/src/app/models/inscripcion.ts
import { Torneo } from './torneo';
import { Usuario } from './usuario';

export interface Inscripcion {
  id: string;
  idTorneo: string;
  idParticipante: string;
  horaInscripcion: string;
  fecha: string;
  estado: string;
  participante: Usuario;
  torneo: Torneo;
}
